user-guide/Getting_started/Installing_a_DMA/Using_a_pre_installed_DataMiner_Virtual_Hard_Disk.md

```diff
@@ -127,6 +127,51 @@ Follow the below steps to configure your DataMiner Agent:
 > [!IMPORTANT]
 > For security reasons, we strongly recommend creating a second user and disabling the built-in administrator account once the setup is complete. See [Managing users](xref:Managing_users).
 
+## WSL management
+
+If you selected the option *Self-hosted - Local storage*, which is not recommended and should only be done for **testing and staging environments**, that means you are running a WSL container for your storage.
+
+If you want to manage the configuration of Cassandra or OpenSearch for this setup, you should **connect to the container**. To do so, you will need to open a command prompt with the Administrator user (or the user account used for the installation) and run the `wsl` command. This way, you should automatically be logged in and connected to the WSL container.
+
+> [!NOTE]
+> WSL always runs under a specific user. It is not possible to reach a WSL container from another user account.
```

Comment:
Does this mean that if a different account than the Administrator account was used, it won't be possible to use the Administrator account?